test/e2e/multikueue/suite_test.go

```diff
@@ -47,6 +56,111 @@ var (
 	ctx              context.Context
 )
 
+func policyRule(group, resource string, verbs ...string) rbacv1.PolicyRule {
+	return rbacv1.PolicyRule{
+		APIGroups: []string{group},
+		Resources: []string{resource},
+		Verbs:     verbs,
+	}
+}
+
+func kubeconfigForMultiKueueSA(ctx context.Context, c client.Client, restConfig *rest.Config, ns string, prefix string, clusterName string) ([]byte, error) {
```

Comment:
Let's take a chance to add a comment on this function for all the things we are creating here.

Reply:
done